src/world.ts

```diff
@@ -107,18 +120,15 @@ class World {
     }
 
     /**
-        Reset world data to a barren world with invaders and source keepers users
+        Reset world data to a barren world with no rooms, but with invaders and source keepers users
     */
     async reset() {
         const { db, env } = await this.load();
         // Clear database
         await Promise.all(_.map(db, (col) => col.clear()));
         await env.set('gameTime', 1);
-        // Generate basic terrain data
-        const terrain = new TerrainMatrix();
```

Comment:
This terrain was never actually used.

Was it intended to put random call cells in the room? This function removes all existing rooms anyways so there would not be a location to place it.

Probably better to just clear everything.

Reply:
Looks like good cleanup